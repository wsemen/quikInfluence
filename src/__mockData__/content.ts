export const homepageContent = [
  {
    id: '94a68462-8dc6-4380-aa89-2798925ac39d',
    resource: 'quik-influence',
    page: 'Home Page',
    type: 'info',
    content: {
      sub_header:
        'We craft unique digital experiences, Because of this we have an endless supply of leads all organic and verified.',
      header:
        'Our Specialized apps and services make customer acquisition easy',
      footer:
        'Contact us about custom plans and agreements to ensure we meet your specific needs!',
      cards: [
        {
          id: 1,
          header: 'Industry Experts',
          desc: 'Not sure what your looking for? Our team can help you dial in on what will be your next audience.',
          img: '',
        },
        {
          id: 2,
          header: "100's of industries",
          desc: 'Our leads are sourced from a Multi-Channel system that spans across many different platforms and industries',
          img: '',
        },
        {
          id: 3,
          header: 'Custom experience',
          desc: 'Our amazing development team can create an expereince tailored to your comapnies needs. No matter what.',
          img: '',
        },
      ],
    },
    meta: null,
    createdAt: '2022-02-12T13:02:46.002Z',
    updatedAt: '2022-02-12T16:10:26.437Z',
  },
  {
    id: '4c722eab-31ab-406b-8e6b-7f5a76dadf89',
    resource: 'quik-influence',
    page: 'Home Page',
    type: 'footer',
    content: {
      sub_header: 'Connecting dots one lead at a time.',
      header: 'Ready to find your next audience with us?',
      header_desc:
        'No matter the size of the project, Quik Influence can create a custom service plan for you! And with access to leads in a seeminly limitless field of industries and niches, it’s no wonder we’re the go to for CRM needs.',
    },
    meta: null,
    createdAt: '2022-02-12T11:39:59.787Z',
    updatedAt: '2022-02-12T14:07:51.956Z',
  },
  {
    id: 'f2a13a30-a858-4881-9c03-52d9503c0cb3',
    resource: 'quik-influence',
    page: 'Home Page',
    type: 'banner',
    content: {
      '0': 'Find your customer with Quik Influence',
      sub_header: 'QUIK INFLUENCE THE CAPABLE CRM',
      header: 'Find your customer with Quik Influence',
      header_desc:
        'QuikInfluence is a new cutting-edge media publisher. With its foundation built on partnerships with many of the largest influencers on social media and a network of over a dozen proprietary apps, our data feeds and ad placement options are not found anywhere else. Add in some revolutionary technology collaborations and it’s everything you’ve been waiting for in online marketing! Official launch scheduled for late March 2022, stay turned for more details. Can’t wait until the launch? Please contact our partners at Email Agency for more info on our platform at 561-899-0712 or Amie@emailagency.com',
      image_url: 'url',
    },
    meta: null,
    createdAt: '2022-02-05T13:04:36.041Z',
    updatedAt: '2022-02-12T15:17:27.987Z',
  },
];

export const nav = [
  {
    id: 'e3619b24-0868-4a82-8864-84077289ae05',
    resource: 'quik-influence',
    page: 'Nav',
    type: 'navbar',
    content: {
      navLinks: [
        {
          name: 'Home',
          link: '/',
        },
      ],
    },
    meta: null,
    createdAt: '2022-02-05T08:21:14.031Z',
    updatedAt: '2022-02-06T13:35:36.609Z',
  },
];

export const staticContentData = {
  props: {
    pageContent: homepageContent.reduce(
      (acc: any, cur: any) => ({ ...acc, [cur.type]: cur }),
      {}
    ),
  },
  revalidate: 10,
};
