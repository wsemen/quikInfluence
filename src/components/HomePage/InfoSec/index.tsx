import EditableWrapper from 'components/EditableWrapper';
import Section from '../Section';
import { Box, Flex, Image } from '@chakra-ui/react';

const InfoSec = ({ info }: { info: any }) => {
  return (
    <Section
      background="#fff"
      h="unset"
      padding={{ base: '50px 15px', lg: '70px 15px', xl: '100px 15px' }}
    >
      <Flex
        maxW="1440px"
        margin="auto"
        alignItems="center"
        justifyContent="center"
        h="100%"
        direction="column"
      >
        <EditableWrapper sectionId="header" data={info} sectionName="info">
          <Box
            as="h2"
            color="#333"
            maxW={{ base: '90%', lg: '600px', xl: '736px' }}
            textAlign="center"
            margin="0 auto 10px"
            fontSize={{ base: '25px', lg: '30px', xl: '43px' }}
          >
            {info.content.header}
          </Box>
        </EditableWrapper>

        <EditableWrapper sectionId="sub_header" data={info} sectionName="info">
          <Box
            as="p"
            color="#333"
            textAlign="center"
            margin="0 auto 35px"
            maxW={{ base: '70%', lg: '500px', xl: '611px' }}
          >
            {info.content.sub_header}
          </Box>
        </EditableWrapper>

        <Flex
          justifyContent="space-between"
          marginBottom="32px"
          maxW={1000}
          textAlign="center"
          direction={{ base: 'column', md: 'row' }}
          alignItems="center"
        >
          {info.content.cards.map(
            (
              card: { header: string; desc: string; img: string },
              i: number
            ) => (
              <Flex
                key={`info_box_${i}`}
                border="1px solid #E6E6E6"
                alignItems="center"
                justifyContent="center"
                direction="column"
                p="30px 15px"
                w={{ base: '100%', xl: '32%' }}
                margin={{ base: '20px', xl: 0 }}
                maxW="450px"
              >
                <EditableWrapper
                  sectionId={`cards__${i}__header`}
                  data={info}
                  sectionName="info"
                >
                  <Box as="h3" fontSize={25} color="#333" fontWeight="500">
                    {card.header}
                  </Box>
                </EditableWrapper>

                <Image margin="25px 0" src="/icon.png" w="75px" h="75px" />

                <EditableWrapper
                  sectionId={`cards__${i}__desc`}
                  data={info}
                  sectionName="info"
                >
                  <Box as="p" color="#333">
                    {card.desc}
                  </Box>
                </EditableWrapper>
              </Flex>
            )
          )}
        </Flex>

        <EditableWrapper sectionId="footer" data={info} sectionName="info">
          <Box as="p" fontSize={14} fontWeight="500">
            {info.content.footer}
          </Box>
        </EditableWrapper>
      </Flex>
    </Section>
  );
};

export default InfoSec;
