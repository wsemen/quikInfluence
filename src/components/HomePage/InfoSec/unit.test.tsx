import { render } from '@testing-library/react';
import { staticContentData } from '__mockData__/content';
import InfoSec from '.';

it('renders MainContent component', () => {
  const { container } = render(
    <InfoSec info={staticContentData.props.pageContent.info} />
  );
  expect(container).toMatchSnapshot();
});
