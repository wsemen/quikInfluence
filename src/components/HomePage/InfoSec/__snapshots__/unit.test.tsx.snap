// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MainContent component 1`] = `
<div>
  <section
    class="css-15a0p7b"
  >
    <div
      class="css-p7r2ak"
    >
      <div
        class="css-79elbk"
      >
        <div
          class="css-1lx80g1"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-x-ray fa-w-20 "
            data-icon="x-ray"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 20px;"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
              fill="currentColor"
            />
          </svg>
        </div>
        <h2
          class="css-1tpxf1l"
          contenteditable="false"
          id="info_header"
          tabindex="-1"
        >
          Our Specialized apps and services make customer acquisition easy
        </h2>
      </div>
      <div
        class="css-79elbk"
      >
        <div
          class="css-1lx80g1"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-x-ray fa-w-20 "
            data-icon="x-ray"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 20px;"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
              fill="currentColor"
            />
          </svg>
        </div>
        <p
          class="css-1h356hx"
          contenteditable="false"
          id="info_sub_header"
          tabindex="-1"
        >
          We craft unique digital experiences, Because of this we have an endless supply of leads all organic and verified.
        </p>
      </div>
      <div
        class="css-b69u32"
      >
        <div
          class="css-9chr40"
        >
          <div
            class="css-79elbk"
          >
            <div
              class="css-1lx80g1"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-x-ray fa-w-20 "
                data-icon="x-ray"
                data-prefix="fas"
                focusable="false"
                role="img"
                style="width: 20px;"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <h3
              class="css-1mws978"
              contenteditable="false"
              id="info_cards__0__header"
              tabindex="-1"
            >
              Industry Experts
            </h3>
          </div>
          <img
            class="chakra-image css-20teaf"
            src="/icon.png"
          />
          <div
            class="css-79elbk"
          >
            <div
              class="css-1lx80g1"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-x-ray fa-w-20 "
                data-icon="x-ray"
                data-prefix="fas"
                focusable="false"
                role="img"
                style="width: 20px;"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <p
              class="css-1yxu09j"
              contenteditable="false"
              id="info_cards__0__desc"
              tabindex="-1"
            >
              Not sure what your looking for? Our team can help you dial in on what will be your next audience.
            </p>
          </div>
        </div>
        <div
          class="css-9chr40"
        >
          <div
            class="css-79elbk"
          >
            <div
              class="css-1lx80g1"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-x-ray fa-w-20 "
                data-icon="x-ray"
                data-prefix="fas"
                focusable="false"
                role="img"
                style="width: 20px;"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <h3
              class="css-1mws978"
              contenteditable="false"
              id="info_cards__1__header"
              tabindex="-1"
            >
              100's of industries
            </h3>
          </div>
          <img
            class="chakra-image css-20teaf"
            src="/icon.png"
          />
          <div
            class="css-79elbk"
          >
            <div
              class="css-1lx80g1"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-x-ray fa-w-20 "
                data-icon="x-ray"
                data-prefix="fas"
                focusable="false"
                role="img"
                style="width: 20px;"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <p
              class="css-1yxu09j"
              contenteditable="false"
              id="info_cards__1__desc"
              tabindex="-1"
            >
              Our leads are sourced from a Multi-Channel system that spans across many different platforms and industries
            </p>
          </div>
        </div>
        <div
          class="css-9chr40"
        >
          <div
            class="css-79elbk"
          >
            <div
              class="css-1lx80g1"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-x-ray fa-w-20 "
                data-icon="x-ray"
                data-prefix="fas"
                focusable="false"
                role="img"
                style="width: 20px;"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <h3
              class="css-1mws978"
              contenteditable="false"
              id="info_cards__2__header"
              tabindex="-1"
            >
              Custom experience
            </h3>
          </div>
          <img
            class="chakra-image css-20teaf"
            src="/icon.png"
          />
          <div
            class="css-79elbk"
          >
            <div
              class="css-1lx80g1"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-x-ray fa-w-20 "
                data-icon="x-ray"
                data-prefix="fas"
                focusable="false"
                role="img"
                style="width: 20px;"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <p
              class="css-1yxu09j"
              contenteditable="false"
              id="info_cards__2__desc"
              tabindex="-1"
            >
              Our amazing development team can create an expereince tailored to your comapnies needs. No matter what.
            </p>
          </div>
        </div>
      </div>
      <div
        class="css-79elbk"
      >
        <div
          class="css-1lx80g1"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-x-ray fa-w-20 "
            data-icon="x-ray"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="width: 20px;"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M240 384c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm160 32c8.8 0 16-7.2 16-16s-7.2-16-16-16-16 7.2-16 16 7.2 16 16 16zM624 0H16C7.2 0 0 7.2 0 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16zm0 448h-48V96H64v352H16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h608c8.8 0 16-7.2 16-16v-32c0-8.8-7.2-16-16-16zM480 248c0 4.4-3.6 8-8 8H336v32h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h64c26.5 0 48 21.5 48 48s-21.5 48-48 48-48-21.5-48-48v-16h-64v16c0 26.5-21.5 48-48 48s-48-21.5-48-48 21.5-48 48-48h64v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-32H168c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h136v-32H200c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h104v-24c0-4.4 3.6-8 8-8h16c4.4 0 8 3.6 8 8v24h104c4.4 0 8 3.6 8 8v16c0 4.4-3.6 8-8 8H336v32h136c4.4 0 8 3.6 8 8v16z"
              fill="currentColor"
            />
          </svg>
        </div>
        <p
          class="css-bg3st0"
          contenteditable="false"
          id="info_footer"
          tabindex="-1"
        >
          Contact us about custom plans and agreements to ensure we meet your specific needs!
        </p>
      </div>
    </div>
  </section>
</div>
`;
